crates/girlmath/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: test-doc
+test:
+	cargo test --doc --package girlmath
```

Comment:
Feel free to ignore this, but I have an aversion to Makefiles as a command runner for some reason and tend to use `just`. Here's a Justfile with a few good commands for this project.

```make
test-rust:
    cargo test --doc --package girlmath

test-js:
    npm --prefix bindings/girlmath-js run test

build-wasm:
    npm --prefix bindings/girlmath-js run build
```

Reply:
does `just` have to be installed?

Reply:
it does `brew install just` or `cargo install just`

Reply:
done https://github.com/TBD54566975/girlmath/commit/7d7f8ec9c5c66d82bca40bf36af390672c4bb47f